kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -618,7 +618,7 @@ impl<'tcx> GotocCtx<'tcx> {
         self.find_function(&fname).unwrap().call(vec![init])
     }
 
-    pub fn codegen_func_expr(&mut self, instance: Instance<'tcx>, span: Option<&Span>) -> Expr {
+    pub fn ensure_func(&mut self, instance: Instance<'tcx>) -> (String, Type) {
```

Comment:
Might be cleaner to just return the symbol, which would have the name and type included

Reply:
Good point, done.

Reply:
Actually, that does not work. Because at least for the function `memcmp`, the signature generated by `codegen_function_sig` is different from the type that's already in the symbol table:
```
$ kani tests/kani/Whitespace/main.rs
[...]
```

Reply:
I did not want to spend too much time on this, so I decided to just return the symbol and the type for now. I filed #1350 to track this problem and added a fixme in the code.